release.py

```diff
@@ -97,21 +97,23 @@ def update_docker_image(new_version, update_latest_tag):
     # build the open source app
     version_build_cmd = 'bazel run -c opt --stamp --define version=%s --define release=true deployment:release_onprem' % new_version
     run_or_die(version_build_cmd)
-    if update_latest_tag:
-        latest_build_cmd = 'bazel run -c opt --stamp --define version=latest --define release=true deployment:release_onprem'
-        run_or_die(latest_build_cmd)
 
     # build the enterprise app
     version_build_cmd = 'bazel run -c opt --stamp --define version=enterprise-%s --define release=true enterprise/deployment:release_enterprise' % new_version
     run_or_die(version_build_cmd)
-    if update_latest_tag:
-        latest_build_cmd = 'bazel run -c opt --stamp --define version=latest --define release=true enterprise/deployment:release_enterprise'
-        run_or_die(latest_build_cmd)
 
     # build the executor
     version_build_cmd = 'bazel run -c opt --stamp --define version=enterprise-%s --define release=true enterprise/deployment:release_executor_enterprise' % new_version
     run_or_die(version_build_cmd)
+
+    # update "latest" tags
     if update_latest_tag:
+        latest_build_cmd = 'bazel run -c opt --stamp --define version=latest --define release=true deployment:release_onprem'
+        run_or_die(latest_build_cmd)
+
+        latest_build_cmd = 'bazel run -c opt --stamp --define version=latest --define release=true enterprise/deployment:release_enterprise'
+        run_or_die(latest_build_cmd)
```

Comment:
nit: call these vars 2 diff things to avoid silly typo errors later

Reply:
fixed here and above